src/lib.rs

```diff
@@ -382,8 +395,8 @@ pub trait PolynomialCommitment<G: AffineCurve>: Sized {
         let rands_iter = labeled_randomnesses.into_iter();
```

Comment:
Would not be equivalent to define  has_hiding = ! labeled_randomness.is_empty() ?
The commitment is non-hiding exactly when there is no randomness.